test/unit/firebase-app.spec.ts

```diff
@@ -159,44 +161,53 @@ describe('FirebaseApp', () => {
     it('should ammend the missing config values with the ones in the config file', () => {
       process.env[FIREBASE_CONFIG_FILE_VAR] = './test/resources/firebase_config.json';
       const app = firebaseNamespace.initializeApp(mocks.appOptionsNoDatabaseUrl, mocks.appName);
-      expect(app.options.databaseURL).to.equal("https://hipster-chat.firebaseio.mock");
-      expect(app.options.projectId).to.equal("hipster-chat-mock");
-      expect(app.options.storageBucket).to.equal("hipster-chat.appspot.mock");
+      expect(app.options.databaseAuthVariableOverride).to.equal('this#is#an#auth#string');
+      expect(app.options.databaseURL).to.equal('https://hipster-chat.firebaseio.mock');
+      expect(app.options.projectId).to.equal('hipster-chat-mock');
+      expect(app.options.storageBucket).to.equal('hipster-chat.appspot.mock');
     });
 
     it('should throw when the environment variable points to non existing file', () => {
       process.env[FIREBASE_CONFIG_FILE_VAR] = './test/resources/non_existant.json';
-      
+
       expect(() => {
-       const app = firebaseNamespace.initializeApp(mocks.appOptionsNoDatabaseUrl, mocks.appName);
-      }).to.throw(`Failed to read app options file: Error: ENOENT: no such file or directory, open './test/resources/non_existant.json'`);
+       firebaseNamespace.initializeApp(mocks.appOptionsNoDatabaseUrl, mocks.appName);
```

Comment:
indent one more space so you are indented by 2 spaces from encapsulating `expect`

Reply:
done